code/modules/mining/mine_turfs.dm

```diff
@@ -70,30 +70,30 @@ var/list/mineral_can_smooth_with = list(
 
 // Copypaste parent call for performance.
 /turf/simulated/mineral/Initialize(mapload)
-	if (initialized)
+	if(initialized)
 		crash_with("Warning: [src]([type]) initialized multiple times!")
 
-	if (icon != actual_icon)
+	if(icon != actual_icon)
 		icon = actual_icon
 
 	initialized = TRUE
 
 	turfs += src
 
 	if(dynamic_lighting)
-		luminosity = 0
+		luminosity = FALSE
```

Comment:
luminosity is not a boolean:
http://www.byond.com/docs/ref/#/atom/var/luminosity